cfgov/regulations3k/management/commands/update_regulation_index.py

```diff
@@ -9,15 +9,15 @@
 logger = logging.getLogger(__name__)
 
 
-def _run_haystack_update():
-    """Update the Haystack index after prepping section paragraphs."""
-    call_command('update_index', 'regulations3k', '--remove')
+def _run_elasticsearch_rebuild():
+    """Rebuild the Elasticsearch index after prepping section paragraphs."""
+    call_command('search_index', 'regulations3k', '--rebuild -f --parallel')
```

Comment:
Indexing one model in DSL requires a `--models` arg, and since it's one app, we probably don't need parallel.

```suggestion
    call_command('search_index', '--rebuild', '-f', '--models', 'regulations3k')
```

Reply:
Added `--models` argument and removed `parallel` argument